src/service/ampdoc-impl.js

```diff
@@ -300,6 +303,14 @@ export class AmpDoc {
   contains(node) {
     return this.getRootNode().contains(node);
   }
+
+  /**
+   * Binding of 3p analytics vendors' responses, used amp-ad-exit
+   * @return {!Object<string, !Object<string, Object<string,string>>>}
+   */
+  getIframeTransportResponses() {
```

Comment:
Initially, I put it in anchor-click-interceptor, and this didn't work. That's because the publisher page and the creative each request different JS source files, due to being bundled differently. So, I had to put it in the AmpDoc instance for the parent page. I agree that it's ugly - if you know of a way around this, please let me know.